ui/app/components/ui/list-item/list-item.component.js

```diff
@@ -27,9 +27,7 @@ export default function ListItem({
         {React.isValidElement(title) ? (
           title
         ) : (
-          <button onClick={onClick}>
```

Comment:
There were existing rules for `list-item__title` regarding text truncation, but being the direct child of a button tag was blocking them taking effect. This button element and associated `onClick` value are actually redundant, as the  list item wrapper handles the `onClick` event for all of its contents: https://github.com/MetaMask/metamask-extension/blob/187c7828cfe1402e2e905924f2c29304ccee14d4/ui/app/components/ui/list-item/list-item.component.js#L22

Reply:
Huh, interesting find!

The use of `button` here was important though - it allowed keyboard navigation. Though... now that I'm testing it... it seems to have made it focusable, but pressing "Enter" did nothing :disappointed:.

Well, given that this already wasn't keyboard-friendly, this doesn't need to block here. But that's something we'll want to do. I'd suggest changing the `div` with the `onClick` set to a button.

Reply:
Suggestion makes sense here 👍  Updated the outer container to be of type `button`:

<img width="356" alt="Screen Shot 2021-03-08 at 10 26 53 AM" src="https://user-images.githubusercontent.com/8732757/110358357-892dda80-7ff9-11eb-8637-9e5a8e56f784.png">